File: reddio-csharp/reddio.api/V1/Rest/GetVaultIdMessage.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Reddio.Api.V1.Rest
{
    public class GetVaultIdMessage
    {
        [JsonProperty("asset_id")] public string AssetId { get; set; }
        [JsonProperty("stark_keys")] public List<string> StarkKeys { get; set; }

        public GetVaultIdMessage()
        {
        }

        public GetVaultIdMessage(string assetId, List<string> starkKeys)
        {
            AssetId = assetId;
            StarkKeys = starkKeys;
        }
    }
}

File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/GetVaultIdMessage.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class GetVaultIdMessage {
    @JsonProperty("asset_id")
    private String assetId;
    @JsonProperty("stark_keys")
    private List<String> starkKeys;
}
